android/app/src/main/java/com/dismoi/scout/accessibility/BackgroundService.kt

```diff
@@ -115,4 +138,50 @@ class Activity : AccessibilityService() {
     browsers.add(SupportedBrowserConfig("com.android.chrome", "com.android.chrome:id/url_bar"))
     return browsers
   }
+
+  private fun createNotificationChannel() {
+    // Create the NotificationChannel, but only on API 26+ because
+    // the NotificationChannel class is new and not in the support library
```

Comment:
Then could you do some kind of feature detection instead? Instead of testing the version, for example, maybe you could check if the class exists?

Reply:
These conditions are in autocorrect mode with android studio. But android studio also proposes decorators on the top of the function to handle these kind on conditions. Would you rather having decorators?